app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :likes, dependent: :destroy
   has_many :comments, dependent: :destroy
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   # Registers a callback to be called after a record is created
   after_create :update_post_counter
```

Comment:
- [ ] Please validate the input text as well as fields.

![setup-3](https://user-images.githubusercontent.com/55635977/145416355-d5780947-de74-4609-af9a-6c9140bb7197.png)